packages/dynamoose/lib/Table/index.ts

```diff
@@ -554,13 +556,18 @@ export enum TableUpdateOptions {
 	throughput = "throughput",
 	tags = "tags",
 	tableClass = "tableClass",
-	streams = "streams"
+	streams = "streams",
+	replication = "replication"
```

Comment:
Did you update the JSDoc to include this?  On the `constructor` documentation there is a section for table update options. It has some information about this.